bin/sr-run-ci-build.sh

```diff
@@ -23,7 +23,7 @@ if  [ "semaphore_docker" != $server_type ] && [ "local" != $server_type ] && [ "
   sudo apt-get update
   sudo apt-get install -y python-dev libxml2-dev libxslt-dev python-pip lcov wget git libssl-dev libffi-dev libyaml-dev
   # Temporary fix for https://github.com/pypa/pip/issues/7217
```

Comment:
This comment should be deleted, right?

Reply:
Ugh, yes. Fixed in https://github.com/shadow-robot/sr-build-tools/pull/555/commits/0d62672567af37f6c43672ab2876b3b4d0da1091